src/views/instancetypes/list/InstanceTypePage.tsx

```diff
@@ -23,16 +25,21 @@ import '@kubevirt-utils/styles/list-managment-group.scss';
 
 const InstanceTypePage: FC<ListPageProps> = (props) => {
   const { t } = useKubevirtTranslation();
+  const isAdmin = useIsAdmin();
   const navigate = useNavigate();
   const location = useLocation();
-  const activeNamespace = props?.namespace ?? ALL_NAMESPACES;
+  const [activeNamespace] = useActiveNamespace();
+  const currentNamespace = props?.namespace ?? activeNamespace ?? ALL_NAMESPACES;
 
   const isSearchPage = useIsSearchPage();
 
   const activeTabKey = useMemo(
-    () => (location?.pathname.includes(VirtualMachineClusterInstancetypeModel.kind) ? 0 : 1),
+    () =>
+      location?.pathname.includes(VirtualMachineClusterInstancetypeModel.kind) && isAdmin ? 0 : 1,
```

Comment:
why do we need the isAdmin here too? won't this prevent from non-admin users to reach the VM create flow with IT?